components/suggest/src/schema.rs

```diff
@@ -26,14 +26,21 @@ pub const SQL: &str = "
     CREATE TABLE suggestions(
         id INTEGER PRIMARY KEY,
         record_id TEXT NOT NULL,
-        block_id INTEGER NOT NULL,
         advertiser TEXT NOT NULL,
-        iab_category TEXT NOT NULL,
         title TEXT NOT NULL,
         url TEXT NOT NULL,
+        icon_id TEXT NOT NULL
+    );
+
+    CREATE TABLE adm_custom_details(
+        suggestion_id INTEGER,
+        block_id INTEGER,
+        advertiser TEXT NOT NULL,
+        iab_category TEXT NOT NULL,
         icon_id TEXT NOT NULL,
         impression_url TEXT,
-        click_url TEXT
+        click_url TEXT,
+        FOREIGN KEY(suggestion_id) REFERENCES suggestions(id)
```

Comment:
I can't remember how suggestions are pruned and I'm too lazy to look, but I wonder if an "on cascade delete" should happen?